source/_integrations/image.markdown

```diff
@@ -23,3 +23,35 @@ In addition, the entity can have the following states:
 
 - **Unavailable**: The entity is currently unavailable.
 - **Unknown**: The state is not yet known.
+
+### Actions
+
+Once loaded, the `image` platform will expose services that can be called to perform various actions.
+
+Available services: `snapshot`.
+
+#### Action `snapshot`
+
+Take a snapshot from an image.
+
+| Service data attribute | Optional | Description |
+| ---------------------- | -------- | ----------- |
+| `entity_id`            |      no  | Name(s) of entities to create a snapshot from, e.g., `image.my_image`. |
+| `filename`             |      no  | Template of a file name. Variable is `entity_id`, e.g., {% raw %}`/tmp/snapshot_{{ entity_id.name }}`{% endraw %}. |
```

Comment:
The entity_id variable was removed. The filename parameter is not always a template. Action parameters can always be templates. We don't need to mention that here.